test/05-device-state.spec.ts

```diff
@@ -406,46 +379,58 @@ describe('deviceState', () => {
-			// Copy previous values to restore later
-			const previousValue = deviceState.applyInProgress;
-			// Setup failing conditions
-			deviceState.applyInProgress = true;
-			expect(await deviceState.healthcheck()).to.equal(false);
-			expect(Log.info).to.be.calledOnce;
-			expect((Log.info as SinonSpy).lastCall?.lastArg).to.equal(
-				stripIndent`
-				Healthcheck failure - At least ONE of the following conditions must be true:
-					- No applyInProgress      ? false
-					- fetchesInProgress       ? false
-					- cycleTimeWithinInterval ? false`,
-			);
-			// Restore value
-			deviceState.applyInProgress = previousValue;
-		});
+		// it.skip('passes with correct conditions', async () => {
+		// 	// Setup passing condition
+		// 	const previousValue = deviceState.__get__('applyInProgress');
+		// 	deviceState.__set__('applyInProgress', false);
+
+		// 	// const previousValue = deviceState.applyInProgress;
+		// 	// deviceState.applyInProgress = false;
+
+		// 	expect(await deviceState.healthcheck()).to.equal(true);
+
+		// 	// Restore value
+		// 	deviceState.__set__('applyInProgress', previousValue);
+		// 	// deviceState.applyInProgress = previousValue;
+		// });
+
+		// it.skip('passes if unmanaged is true and exit early', async () => {
+		// 	// Setup failing conditions
+		// 	const previousPollTime = deviceState.__get__('maxPollTime');
+		// 	const previousValue = deviceState.__get__('applyInProgress');
+		// 	deviceState.__set__('applyInProgress', true);
+		// 	deviceState.__set__('maxPollTime', 1);
+		// 	// Verify this causes healthcheck to fail
+		// 	expect(await deviceState.healthcheck()).to.equal(false);
+		// 	// Do it again but set unmanaged to true
+		// 	configStub.resolves({
+		// 		unmanaged: true,
+		// 	});
+		// 	expect(await deviceState.healthcheck()).to.equal(true);
+		// 	// Restore value
+		// 	deviceState.__set__('applyInProgress', previousValue);
+		// 	deviceState.__set__('maxPollTime', previousPollTime);
+		// });
+
+		// it.skip('fails when applyTargetHealthy is false', async () => {
+		// 	// Copy previous values to restore later
+		// 	const previousValue = deviceState.__get__('applyInProgress');
+		// 	const previousPollTime = deviceState.__get__('maxPollTime');
+		// 	// Setup failing conditions
+		// 	deviceState.__set__('applyInProgress', true);
+		// 	deviceState.__set__('maxPollTime', 1);
+		// 	expect(await deviceState.healthcheck()).to.equal(false);
+		// 	expect(Log.info).to.be.calledOnce;
+		// 	expect((Log.info as SinonSpy).lastCall?.lastArg).to.equal(
+		// 		stripIndent`
+		// 		Healthcheck failure - Atleast ONE of the following conditions must be true:
+		// 			- No applyInProgress      ? false
+		// 			- fetchesInProgress       ? false
+		// 			- cycleTimeWithinInterval ? false`,
+		// 	);
+		// 	// Restore value
+		// 	deviceState.__set__('applyInProgress', previousValue);
+		// 	deviceState.__set__('maxPollTime', previousPollTime);
+		// });
```

Comment:
What's happening with this commented code?

Reply:
It needs dropping out, good spot.